packages/integrations/useAxios/index.ts

```diff
@@ -8,10 +20,22 @@ import axios, { AxiosError, AxiosRequestConfig, AxiosResponse, CancelTokenSource
  * @param url
  * @param config
  */
-export function useAxios<T = any>(
-  url: string,
-  config?: AxiosRequestConfig,
-) {
+export function useAxios<T = any>(url: string, ...args: any[]) {
+  let config: AxiosRequestConfig = {}
+  let instance: AxiosInstance = axios
+
+  if (args.length > 0) {
+    if ('request' in args[0])
```

Comment:
This is a safe way to detect whether it's an axios instance?

Reply:
I was going to use `instanceof` but it looks like you can't do that(https://github.com/axios/axios/issues/737) so I used the way that was suggested here. It doesn't seem to have any overlap with the `AxiosRequestConfig`, so it works from my testing. But I guess the only real way to enforce it is with the typescript types for parameters. If you have any ideas on another way to do it let me know.

Reply:
Alright, go with it then. Can you add a comment above that line mentioning about the issue you linked? Thanks.

Reply:
Sure thing!